upload_worldbank_data/Dockerfile

```diff
@@ -22,7 +24,7 @@ RUN mkdir -p /opt/$NAME/data
 WORKDIR /opt/$NAME/
 COPY contents .
 
-RUN useradd -r $NAME
+RUN adduser $NAME
```

Comment:
Do you know why we were specifying `-r` ? Based on https://linux.die.net/man/8/useradd that creates a "system" user. It looks like most of our docker containers do this but not all.